stress/basic.go

```diff
@@ -300,22 +318,22 @@ func (c *BasicClient) Batch(ps <-chan Point, r chan<- response) error {
 
 		if ctr%c.BatchSize == 0 && ctr != 0 {
 			b := buf.Bytes()
-
+			if len(b) == 0 {
+				continue
+			}
 			// Trimming the trailing newline character
 			b = b[0 : len(b)-1]
 
-			wg.Add(1)
-			counter.Increment()
-			go func(byt []byte) {
-				c.retry(byt, time.Duration(1))
-				counter.Decrement()
-				wg.Done()
-			}(b)
-
+			writeBatch(b)
 			var temp bytes.Buffer
 			buf = temp
 		}
 	}
+	// Write out any remaining points
+	b := buf.Bytes()
+	if len(b) > 0 {
+		writeBatch(b)
+	}
```

Comment:
This change makes its so that if your total point count is not a multiple of your batch size the last points still get written.